lib/Traits/DiscretelyTimeVaryingTraits.ts

```diff
@@ -13,10 +13,7 @@ export default class DiscretelyTimeVaryingTraits extends mixTraits(
   @primitiveTrait({
     name: "Mapping from Continuous Time",
     description:
-      "Specifies how a continuous time (e.g. the timeline control) is mapped to a discrete time for this dataset. Valid values are:\n\n" +
-      "  * `nearest` - the nearest available discrete time to the current continuous time is used.\n" +
-      "  * `next` - the discrete time equal to or after the current continuous time is used.\n" +
-      "  * `previous` - the discrete time equal to or before the current continuous time is used.",
+      "Specifies how a continuous time (e.g. the timeline control) is mapped to a discrete time for this dataset. Valid values are: <br/> * `nearest` - the nearest available discrete time to the current continuous time is used. <br/> * `next` - the discrete time equal to or after the current continuous time is used. <br/> * `previous` - the discrete time equal to or before the current continuous time is used.",
```

Comment:
Can we put this back on multiple lines for readability?

Reply:
Done